packages/vsce/src/trees/CICSTree.ts

```diff
@@ -112,24 +104,21 @@ export class CICSTree implements TreeDataProvider<CICSSessionTree> {
           ),
           {
             ignoreFocusOut: true,
-            placeHolder: "Load Profile or Create New Profile",
+            placeHolder: "Edit Team Configuration File",
           }
         );
         if (profileNameToLoad) {
           // If Create New CICS Profile option chosen
-          if (profileNameToLoad.label.includes("\uFF0B")) {
-            if (configInstance.getTeamConfig().exists) {
-              // get all profiles of all types including zosmf
-              const profiles = configInstance.getAllProfiles();
-              if (!profiles.length) {
-                window.showErrorMessage("No profiles found in config file. Create a new config file or add a profile to get started");
-              }
-              const currentProfile = await ProfileManagement.getProfilesCache().getProfileFromConfig(profiles[0].profName);
-              const filePath = currentProfile?.profLoc.osLoc?.[0] ?? "";
-              await openConfigFile(filePath);
-            } else {
-              await this.createNewProfile();
-            }
+          if (profileNameToLoad.label.includes("\u270F")) {
+            // get all profiles of all types including zosmf
+            const profiles = configInstance.getAllProfiles();
+            const currentProfile =
+                profiles.length > 0 ? await ProfileManagement.getProfilesCache().getProfileFromConfig(profiles[0].profName) : null;
+            const filePath =
+                  currentProfile === null
+                    ? configInstance.getTeamConfig().opts.homeDir + "/zowe.config.json"
+                    : (currentProfile?.profLoc.osLoc?.[0] ?? "");
+            await openConfigFile(filePath);
```

Comment:
It looks like filePath could be set to ""... Have you tested what happens when you call openConfigPath with an empty string? Might be worth reworking that so if can only ever be called with a valid parameter.

Reply:
Resolved..!!!